### PR TITLE
[cross_file] Update README

### PR DESCRIPTION
**Pull request by sidrao2006**
## Pre-launch Checklist

- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated `CHANGELOG.md` to add a description of the change.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the `#hackers-new` channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning

### CODE REVIEW

File: packages/cross_file/README.md [hunk 1]
```diff
@@ -31,4 +31,4 @@ You will find links to the API docs on the [pub page](https://pub.dev/packages/c
 For help getting started with Flutter, view our online
 [documentation](http://flutter.io/).
 
-For help on editing plugin code, view the [documentation](https://flutter.io/platform-plugins/#edit-code).
\ No newline at end of file
+For help on editing plugin code, view the [documentation](https://flutter.io/platform-plugins/#edit-code).
```
Comment (stuartmorgan) [inline packages/cross_file/README.md:34]: As long as you are touching this line, everything from line 29 on is boilerplate that should just be removed.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]
Thanks! This needs a version bump so that it can be published.

File: packages/cross_file/README.md (hunk 1 above)
Comment (sidrao2006) [reply packages/cross_file/README.md:34]: @stuartmorgan, I didn't make any changes to line 34. I'm not really sure what change is git referring to. Do you want me to remove all the boilerplate from line 29 onwards? (I didn't completely understand your comment)

Comment (stuartmorgan) [reply packages/cross_file/README.md:34]: > @stuartmorgan, I didn't make any changes to line 34. I'm not really sure what change is git referring to.

Looks like your editor made a whitespace change.

> Do you want me to remove all the boilerplate from line 29 onwards?

Yes.

**Comment by sidrao2006** [comment]
@stuartmorgan ready for review

**Review by stuartmorgan — APPROVED** [review]
LGTM

**Pull request merged**